JS-Algorithms/challenges.js

```diff
@@ -118,7 +168,21 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let index = 2;
+
+  // f(n) = f(n-1) + f(n-2)  
+  let n1 = 0;
+  let n2 = 1;
+  let fn = 1;
```

Comment:
I think fn can be removed, you can use n2 variable and have the same result right?

Reply:
That's true, I hadn't noticed it.